src/Feed/CalendarApiFeedType.php

```diff
@@ -72,24 +72,35 @@ public function getData(Feed $feed): array
                 return [];
             }
 
-            $resources = $configuration['resources'];
+            $requestedResources = $configuration['resources'];
+
+            $allResources = $this->loadResources();
 
             $events = [];
 
-            foreach ($resources as $resource) {
-                $events += $this->getResourceEvents($resource);
+            foreach ($requestedResources as $requestedResource) {
+                $events += $this->getResourceEvents($requestedResource);
```

Comment:
```suggestion
                $events[] = $this->getResourceEvents($requestedResource);
```

Personal preference but I like this syntax better for arrays. It help communicate the (array) data type

Reply:
Changed to array_merge